index.js

```diff
@@ -182,26 +182,40 @@ bot.on('ready', () => {
+            if (DeadChatQuestion == 6){DCPQuestion = "What is your favorite form of transportation?"};
             if (DeadChatQuestion == 7){DCPQuestion = "What's your favorite food?"};
-            if (DeadChatQuestion == 8){DCPQuestion = "How has quarantine been?"};
-            if (DeadChatQuestion == 9){DCPQuestion = "Windows, Mac or Linux?"};
-            if (DeadChatQuestion == 10){DCPQuestion = "Are you reading this?"};
-            if (DeadChatQuestion == 11){DCPQuestion = "Are you a communist?"};
-            if (DeadChatQuestion == 12){DCPQuestion = "Where you sleeping?"};
-            if (DeadChatQuestion == 13){DCPQuestion = "Are dogs communist?"};
+            if (DeadChatQuestion == 8){DCPQuestion = "Are we seeing signs of evolution in our species?"};
+            if (DeadChatQuestion == 9){DCPQuestion = "Windows, MacOS or Linux?"};
+            if (DeadChatQuestion == 10){DCPQuestion = "Playstation Or Xbox?"};
+            if (DeadChatQuestion == 11){DCPQuestion = "What's your favourite type of music?"};
+            if (DeadChatQuestion == 12){DCPQuestion = "What is your favourite Disney movie?"};
+            if (DeadChatQuestion == 13){DCPQuestion = "Is time relative to a person or universal?"};
             if (DeadChatQuestion == 14){DCPQuestion = "How have you been?"};
             if (DeadChatQuestion == 15){DCPQuestion = "Iphone or Android?"};
             if (DeadChatQuestion == 16){DCPQuestion = "Do you have pets?"};
             if (DeadChatQuestion == 17){DCPQuestion = "Airplane or Car?"};
-            if (DeadChatQuestion == 18){DCPQuestion = "Ping!"};
+            if (DeadChatQuestion == 18){DCPQuestion = "What are you currently working on in Survival?"};
+	    if (DeadChatQuestion == 19){DCPQuestion = "Why is science so important to modern society?"};
+            if (DeadChatQuestion == 20){DCPQuestion = "What are you currently working on in WarLands?};
+	    if (DeadChatQuestion == 21){DCPQuestion = "Laptop, Desktop or Handheld?"};
+	    if (DeadChatQuestion == 22){DCPQuestion = "What is your favorite version of Minecraft?"};
+	    if (DeadChatQuestion == 23){DCPQuestion = "What is the most valuable thing you currently have ingame?"};
+	    if (DeadChatQuestion == 24){DCPQuestion = "What do you like to do on the weekends?"};
+	    if (DeadChatQuestion == 25){DCPQuestion = "What's your first memory?"};
+	    if (DeadChatQuestion == 26){DCPQuestion = "What's the weirdest quirk you find funny?"};
+	    if (DeadChatQuestion == 27){DCPQuestion = "What's the worst thing you ever did as a kid?"};
+	    if (DeadChatQuestion == 28){DCPQuestion = "What trait do you like most about yourself?"};
+	    if (DeadChatQuestion == 29){DCPQuestion = "What song always puts you in a good mood?"};
+	    if (DeadChatQuestion == 30){DCPQuestion = "What movie or book character do you most identify with?"};
+            if (DeadChatQuestion == 30){DCPQuestion = "Do you have pets?"
```

Comment:
Forgot the } after the quote.

Reply:
Whoopsies... yesterday was probably the worst day for me to have done this...